bc_obps/registration/migrations/0112_prod_fix_contacts_post_transfer.py

```diff
@@ -0,0 +1,62 @@
+# Generated by Django 5.0.14 on 2025-05-12 20:10
+
+from django.db import migrations
+
+"""
+One-time forward-only migration to be applied to prod data.
+Purpose: delete operation representatives from operation_contacts table for operations that have been transferred to a new operator.
+"""
+
+
+def remove_contacts_from_transferred_operations(apps, schema_editor):
+    TransferEvent = apps.get_model("registration", "TransferEvent")
+    Contact = apps.get_model("registration", "Contact")
+
+    contacts_count_before = Contact.objects.count()
+
+    # fetch all transfer events
+    transfer_events = TransferEvent.objects.all()
+
+    # iterate over each transfer event, and for each operation in the event, check the operation's contacts
+    for transfer_event in transfer_events:
+        operation = transfer_event.operation
+        from_operator = transfer_event.from_operator
+        to_operator = transfer_event.to_operator
+        operation_contacts = operation.contacts.all()
+
+        operation_contacts_count_before = operation_contacts.count()
+
+        # for each contact in operation_contacts, check if the contact is associated with the from_operator. If so, delete it.
+        for contact in operation_contacts:
+            if contact.operator == from_operator:
+                # log the removal
+                print(
```

Comment:
Yes, I agree, let's pull this out before merging. It was useful for reviewing, though!